lib/api_browser/app/views/directives/attribute_description/member_options.html

```diff
@@ -1,4 +1,4 @@
 <dt>Member type values</dt>
-<dd class="well">
-  <attribute-description attribute="{options: row.value, description: 'These settings apply to the contained type:'}"></attribute-description>
+<dd>
+  <attribute-description attribute="{options: row.value, description: '<em>These settings apply to each element of the collection:</em>'}"></attribute-description>
```

Comment:
Better avoid embedding HTML in the data, and rather provide some data that will do the formatting on the other side.